app/Domain/Tickets/Templates/Submodules/timelineTabs.sub.php

```diff
@@ -0,0 +1,29 @@
+
+
+<?php
+
+function findActive($route)
+{
+    if (str_contains(\Leantime\Core\Frontcontroller::getCurrentRoute(), $route)) {
+        return "active";
+    }
+    return "";
+}
+
+findActive("");
+?>
```

Comment:
This makes me wonder if we can have composers for php templates. I see no reason why we can't. Let's start moving stuff to composers for partials.

On another note though, can this be a blade template?

Reply:
Do we even need composers, the ugly way would be to just rename the file to blade and wrap the whole thing in @php right?

Reply:
or alternatively just replace <?php with @php and @endphp